### ISSUE TITLE
[TF2] [Unintentional Issue?] materials/hud/halloween_bar_surround gets turned green when merasmus goes into hiding

### ISSUE THREAD

**Issue by Whurrhurr** [ValveSoftware/Source-1-Games#4429]
previously, when halloween_bar_surround looked like this
![halloween_bar_surround](https://user-images.githubusercontent.com/33162051/197283831-5fb0ed8a-e9a9-49d7-a922-1b246a0bf057.png)
and the functionally for halloween_bar to turn green when merasmus went into hiding was added, eventually it effected halloween_bar_surround, so they updated the texture to make it not look as obvious, but it's still noticeable
![image](https://user-images.githubusercontent.com/33162051/197285841-be76b077-4540-44a0-a824-4cde487ba060.png)
the issue with this, is that it makes basic retexturing of halloween_bar_surround next to impossible without it looking weird/bad when the whole thing turns green. the only reason i'm making this issue is because i happened to notice it when i was cleaning up halloween_bar_surround and giving it an outer rim/bracket
![halloween_bar_surround_new style 1](https://user-images.githubusercontent.com/33162051/197287165-88ab5e38-f490-4543-acd9-388663b57133.png)

TL;DR: make the "health bar turns green" stuff not affect halloween_bar_surround

**Comment by Whurrhurr** [comment]
updating this, because i finally got around to checking some files in the TF2 SDK, this check here in tf_hud_boss_health.cpp seems to be the cause of the border image turning green: https://github.com/ValveSoftware/source-sdk-2013/blob/68c8b82fdcb41b8ad5abde9fe1f0654254217b8e/src/game/client/tf/tf_hud_boss_health.cpp#L150

while i was originally going to say you could just get rid of it, why not turn it into a client cvar that people can customize?
